packages/tc-schema-sdk/lib/updater.js

```diff
@@ -28,6 +35,7 @@ class SchemaUpdater {
 			schemaData = {
 				schema: {
 					types: readYaml.directory(schemaDirectory, 'types'),
+					relationshipTypes: readRelationshipTypes(schemaDirectory),
```

Comment:
for consistency, could `readYaml.directory` be implemented so it returns `[]` if the directory is missing, then here we could have `relationshipTypes: readYaml.directory(schemaDirectory, 'relationships')`